src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DTCInterfaces/IPrepareInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace System.Transactions.DtcProxyShim.DtcInterfaces;
+
+/// <summary>
+/// The IPrepareInfo interface is superseded by the IPrepareInfo2 interface.
+/// </summary>
+/// <remarks>
+/// See https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms686533(v=vs.85).
```

Comment:
Nit: We strip from `en-us` in docs.microsoft.com links so that clicking the link gets you to the localized content if there is one.

I see that you have done a lot of work copy&pasting msdn docs over to these internal definitions of the interfaces. It is not something that we typically do. The full doc comments are only required on public APIs, but these are internal.

Reply:
Will remove the locale. Yeah, I copy-pasted these since they were helpful to me during development, and may also help in maintaining in the future. If you have an objection to them being there, I can remove.

Reply:
> It is not something that we typically do

The copy&paste copy gets outdated so it is best to just include a link instead of copying the documentation.

Also, coping the docs like this is typically violating the docs license. You are basically re-licensing the docs under a different license. We may get away with it for Microsoft docs since we are Microsoft project, but it would be unacceptable on non-Windows platforms.

Reply:
OK, will remove all docs and leave links only.